utils/s2n_mem.c

```diff
@@ -74,6 +81,7 @@ int s2n_realloc(struct s2n_blob *b, uint32_t size)
         b->data = data;
         b->size = size;
         b->allocated = size;
+        b->growable = true;
```

Comment:
looks like we use 0/1 so can you be consistent with `mlocked`?

Reply:
Done